indexer/indexer.go

```diff
@@ -149,9 +151,9 @@ func defaultBadgerOptions(
 	opts.Compression = options.None
 
 	// Load tables into memory and memory map value logs.
-	opts.TableLoadingMode = options.MemoryMap
-	opts.ValueLogLoadingMode = options.MemoryMap
-
+	opts.TableLoadingMode = options.FileIO
```

Comment:
At some point we should benchmark it that how slow this option is compared to MemoryMap